docs/dev/marketplace_offering.md

```diff
@@ -0,0 +1,111 @@
+- link: "https://www.datadoghq.com/blog/datadog-marketplace/"
+  tag: "Blog"
+  text: "Expand your monitoring reach with the Datadog Marketplace"
+- link: "/developers/integrations/marketplace_offering/"
+  tag: "Documentation"
+  text: "Learn how to develop a Marketplace offering"
+- link: "/developers/datadog_apps/"
+  tag: "Documentation"
+  text: "Learn about Datadog Apps"
+---
+
+## Overview
+
+The Datadog Marketplace is a digital marketplace where Datadog Technology Partners can list their paid offerings to Datadog users.  
+
+While the **Integrations** page includes integrations and custom dashboard widgets built by both Datadog and Technology Partners at no cost, the **Marketplace** page is a commercial platform for Datadog customers and Technology Partners to buy and sell a variety of offerings including Agent-based or API-based integrations, custom dashboard widgets, software subscriptions/licenses, and professional services.
+
+## List an offering 
+
+All Technology Partners can list an out-of-the-box integration on the **Integrations** page, or a commercial paid offering on the **Marketplace** page. The following types of offerings are supported on the Datadog Marketplace:
+
+### Integrations
+Marketplace integrations that submit third-party data or pull data from a user's Datadog account through the [Datadog Agent][15] or the [Datadog API][16]. These integrations can contain a variety of data such as out-of-the-box metrics, events, logs, and more.
+
+### Software licenses
+Software licenses, or subscriptions to SaaS solutions, enable you to deliver and license software solutions to customers through the Datadog Marketplace.
+
+### Custom dashboard widgets
+A custom dashboard widget that adds visual functionality to existing integration data (or other data within Datadog).
+
+### Professional services
+[Professional services][18] enable you to offer your team's services for implementation, support, or management for a set period of time.
+
+## Why join the Datadog Marketplace? 
+
+Marketplace Partners have unique benefits that are not available to partners who list out-of-the-box integrations:
+ 
+  - Go-to-Market motions, including a blog post, quote for a press release, and social media amplification, with access to dedicated sales and marketing resources, focused on accelerating partner growth. 
+
+  - Training and Support for sales enablement in the form of on-demand courses to quickly onboard sales and technical teams.
+
+  - Exclusive opportunities to sponsor conferences and events (such as Datadog DASH) at a discounted rate.
+
+{{< img src="developers/marketplace/marketplace_overview.png" alt="The Datadog Marketplace page" style="width:100%" >}}
+
+## Join the Datadog partner network
+
+Before listing an offering on the Datadog Marketplace, you will first need to apply to the [Datadog Partner Network's][3] **Technology Partner** track. Once your application has been approved, you can begin to develop your offering.
```

Comment:
When need all partners to join the DPN, not just those developing for the Marketplace. 

In these new docs, if a partner is building for the Integrations page, where are we explaining the join DPN, request sandbox, etc. steps?